src/main/java/org/jabref/gui/exporter/DeleteMetadataAction.java

```diff
@@ -0,0 +1,259 @@
+                                                     .appendText("  " + Localization.lang("Skipped - No PDF linked") + ".\n"));
+            } else {
+                for (Path file : files) {
+                    if (Files.exists(file)) {
+                        try {
+                            DeleteMetadataFromFile(file, entry, stateManager.getActiveDatabase().get(), database);
+                            Platform.runLater(
+                                    () -> optionsDialog.getProgressArea().appendText("  " + Localization.lang("OK") + ".\n"));
+                            entriesChanged++;
+                        } catch (Exception e) {
+                            Platform.runLater(() -> {
+                                //lang
+                                optionsDialog.getProgressArea().appendText("  " + Localization.lang("Error while deleting") + " '"
+                                        + file.toString() + "':\n");
+                                optionsDialog.getProgressArea().appendText("    " + e.getLocalizedMessage() + "\n");
+                            });
+                            errors++;
+                        }
+                    } else {
+                        skipped++;
+                        Platform.runLater(() -> {
+                            optionsDialog.getProgressArea()
+                                         .appendText("  " + Localization.lang("Skipped - PDF does not exist") + ":\n");
+                            optionsDialog.getProgressArea().appendText("    " + file.toString() + "\n");
+                        });
+                    }
+                }
+            }
+            if (optionsDialog.isCanceled()) {
+                Platform.runLater(
+                        () -> optionsDialog.getProgressArea().appendText("\n" + Localization.lang("Operation canceled.") + "\n"));
+                break;
+            }
+        }
+        Platform.runLater(() -> {
+            optionsDialog.getProgressArea()
+                         .appendText("\n"
+                                 // lang
+                                 + Localization.lang("Finished deleting metadata for %0 file (%1 skipped, %2 errors).", String
+                                 .valueOf(entriesChanged), String.valueOf(skipped), String.valueOf(errors)));
+            optionsDialog.done();
+        });
+
+        if (!shouldContinue) {
+            return;
+        }
+        dialogService.notify(Localization.lang("Finished deleting metadata for %0 file (%1 skipped, %2 errors).",
+                String.valueOf(entriesChanged), String.valueOf(skipped), String.valueOf(errors)));
+    }
+
+    private void DeleteMetadataFromFile(Path file, BibEntry entry, BibDatabaseContext databaseContext, BibDatabase database) throws Exception {
+        XmpUtilRemover.deleteXmp(file, entry, database, xmpPreferences);
+        embeddedBibExporter.exportToFileByPath(databaseContext, database, filePreferences, file);
```

Comment:
I don't know what this does? `export` does not read as delete. (Don't have the code at hand currently).

Reply:
The ```export```, the second line, is removed and it still have its intended function